lib/options.js

```diff
@@ -20,8 +21,14 @@ function override(options, overrides) {
 }
 
 function loadOptions(fileName, key, options) {
+  var overrides;
+
   if (fs.existsSync(fileName)) {
-    var overrides = JSON.parse(fs.readFileSync(fileName, 'utf8'));
+    try {
+      overrides = require(path.resolve(fileName));
```

Comment:
Isn't this line enough? Requiring .json files works as well, so the catch-case is just trying to do the same thing again.